{
    "comments": [
        {
            "author": "Jukka Zitting",
            "body": "This seems related to JCR-1203. I will take a look at this later today if you don't beat me to it.",
            "date": "2007-11-26T17:08:32.325+0000",
            "id": 0
        },
        {
            "author": "Jukka Zitting",
            "body": "Fixed in revision 598488.",
            "date": "2007-11-27T01:34:46.117+0000",
            "id": 1
        }
    ],
    "component": "jackrabbit-core, test",
    "description": "console output: \n\n-------------------------------------------------------------------------------\nTest set: org.apache.jackrabbit.core.integration.PrepareTestRepository\n-------------------------------------------------------------------------------\nTests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 3.428 sec <<< FAILURE!\ntestPrepareTestRepository(org.apache.jackrabbit.core.integration.PrepareTestRepository)  Time elapsed: 3.397 sec  <<< ERROR!\njavax.jcr.RepositoryException: Invalid node type definition: {http://www.apache.org/jackrabbit/test}versionable already exists: {http://www.apache.org/jackrabbit/test}versionable already exists\n\tat org.apache.jackrabbit.core.nodetype.NodeTypeManagerImpl.registerNodeTypes(NodeTypeManagerImpl.java:308)\n\tat org.apache.jackrabbit.core.nodetype.NodeTypeManagerImpl.registerNodeTypes(NodeTypeManagerImpl.java:488)\n\tat org.apache.jackrabbit.core.integration.PrepareTestRepository.testPrepareTestRepository(PrepareTestRepository.java:49)\n",
    "hasPatch": false,
    "hasScreenshot": false,
    "id": "JCR-1235",
    "issuetypeClassified": "BUG",
    "issuetypeTracker": "BUG",
    "priority": "Minor",
    "product": "JACKRABBIT",
    "project": "JACKRABBIT",
    "summary": "o.a.j.core.integration.PrepareTestRepository fails on 2nd and every subsequent invocation",
    "systemSpecification": true,
    "version": ""
}